pennylane/data/data_manager/__init__.py

```diff
@@ -402,69 +424,58 @@ def remove_paths(foldermap):
     return remove_paths(_get_foldermap())
 
 
-def list_attributes(data_name):
-    r"""List the attributes that exist for a specific ``data_name``.
+def _interactive_request_data_name(data_names):
+    """Prompt the user to select a data name."""
+    print("Please select the data name from the following:")
+    for i, option in enumerate(data_names):
+        print(f"{i + 1}: {option}")
+    choice = input("Choice of data name: ").strip()
+    if choice not in data_names:
+        raise ValueError(f"Must select a single data name from {data_names}")
+    return choice
 
-    Args:
-        data_name (str): The type of the desired data
 
-    Returns:
-        list (str): A list of accepted attributes for a given data name
+def _interactive_request_attributes(attribute_options):
+    """Prompt the user to select a list of attributes."""
+    print(
+        'Please select a list of attributes from the following available attributes or "full" for all attributes.'
+    )
+    for i, option in enumerate(attribute_options):
+        print(f"{i + 1}: {option}")
 
-    .. seealso:: :func:`~.load_interactive`, :func:`~.list_datasets`, :func:`~.load`.
+    choice_input = input("Choice of attributes: ")
+    if isinstance(choice_input, str):
+        choice = choice_input.strip()
+    elif isinstance(choice_input, list):
+        choice = [ci.strip() for ci in choice_input]
```

Comment:
This doesn't work as expected - I couldn't get a list input to work. We should just split the input on commas:
```suggestion
    choice_input = input("Comma-separated list of attributes: ")
    choices = [choice.strip() for choice in choice_input.split(",")]

    if "full" in choices:
        return attribute_options
    if not (choices and set(choices).issubset(set(attribute_options))):
        raise ValueError(f"Must select a list of attributes from {attribute_options}")
    
    return choices
```

Reply:
Thanks yeah this was subtle, it's taking the user's list input in as a string, we needed to also strip the square brackets from the input.